aip/_journals.tab

```diff
@@ -1,4 +1,6 @@
 Title	ISSN	eISSN	Field	Field2
+Journal of Vacuum Science &amp; Technology A: Vacuum, Surfaces, and Films 0734-2101 x physics chemistry
+Journal of Vacuum Science &amp; Technology B: Microelectronics and Nanometer Structures 1520-8567 x physics chemistry
```

Comment:
@rmzelle 
1. +2. noted for the future.
I did use tabs for certain. not sure what happened. Let's see what happens the next time. sorry for the inconvenience and double checking everything.